frontends/web/src/routes/new-settings/about.tsx

```diff
@@ -0,0 +1,25 @@
+import { useTranslation } from 'react-i18next';
+import { Main, Header } from '../../components/layout';
+import { View, ViewContent } from '../../components/view/view';
+import { WithSettingsTabs } from './components/tabs';
+import { AppVersion } from './components/about/app-version-setting';
+
+type TProps = {
+    deviceIDs: string[]
+  }
```

Comment:
use 2 spaces, also do you know if there is a way to tell eslint to also parse typescript?